packages/frontend-2/lib/workspaces/composables/projectUpdates.ts

```diff
@@ -0,0 +1,85 @@
+import { useApolloClient, useSubscription } from '@vue/apollo-composable'
+import { graphql } from '~/lib/common/generated/gql'
+import { WorkspaceProjectsUpdatedMessageType } from '~/lib/common/generated/gql/graphql'
+import { getCacheId, modifyObjectField } from '~/lib/common/helpers/graphql'
+import { ToastNotificationType, useGlobalToast } from '~/lib/common/composables/toast'
+import { projectRoute } from '~/lib/common/helpers/route'
+
+export function useWorkspaceProjectsSubscription(workspaceSlug: ComputedRef<string>) {
```

Comment:
Name could be improved, the convention so far would say "useWorkspaceProjectsUpdatedTracking" or "useOnWorkspaceProjectsUpdated"